Taskfile.yml

```
@@ -4,6 +4,37 @@ includes:
   dist: ./DistTasks.yml
 
 tasks:
+  poetry:install-deps:
+    desc: Install dependencies managed by Poetry
+    cmds:
+      - poetry install --no-root
+
+  docs:serve:
+    desc: Run website locally
+    deps:
+      - task: poetry:install-deps
```

```suggestion
      - task: poetry:install-deps
      - task: docs:gen:commands
```
It's still necessary to generate the command line docs before serving.

---

Ups, forgot about that